### PR TITLE
chore(nat): add vpc_id attribute to the private nat gateway response body

### PR DESCRIPTION
**Pull request by ruwenqiang123**
**What this PR does / why we need it**:
add vpc_id attribute to the private nat gateway response body
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format,
  will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: openstack/nat/v3/gateways/requests.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ type CreateOpts struct {
 }
 
 // DownLinkVpc is an object that represents the subnet configuration to which private NAT gateway belongs.
-type DownLinkVpc struct {
+type DownLinkVpcOpt struct {
```
Comment (Lance52259) [inline openstack/nat/v3/gateways/requests.go:36]:
```suggestion
type DownLinkVpc struct {
```
changing this name will cause code incompatibility.

File: openstack/nat/v3/gateways/results.go [hunk 2]
```diff
@@ -31,14 +31,20 @@ type Gateway struct {
...
 	// The key/value pairs to associate with the NAT geteway.
 	Tags []tags.ResourceTag `json:"tags"`
 	// The enterprise project ID to which the private NAT gateway belongs.
 	EnterpriseProjectId string `json:"enterprise_project_id"`
 }
 
+// DownLinkVpc is an object that represents the VPC configuration to which private NAT gateway belongs.
+type DownLinkVpcResp struct {
+	SubnetId string `json:"virsubnet_id"`
+	VpcId    string `json:"vpc_id"`
```
Comment (Lance52259) [inline openstack/nat/v3/gateways/results.go:45]:
```suggestion
	// ......
	SubnetId string `json:"virsubnet_id"`
	// ......
	VpcId    string `json:"vpc_id"`
```
All comments of parameters are missing.

File: openstack/nat/v3/gateways/requests.go (hunk 1 above)
Comment (ruwenqiang123) [reply openstack/nat/v3/gateways/requests.go]: done

File: openstack/nat/v3/gateways/results.go (hunk 2 above)
Comment (ruwenqiang123) [reply openstack/nat/v3/gateways/results.go]: done

File: openstack/nat/v3/gateways/results.go [hunk 3]
```diff
@@ -31,14 +31,22 @@ type Gateway struct {
...
-	DownLinkVpcs []DownLinkVpc `json:"downlink_vpcs"`
+	// The VPC configuration of the private NAT gateway.
+	DownLinkVpcs []DownLinkVpcResp `json:"downlink_vpcs"`
 	// The key/value pairs to associate with the NAT geteway.
 	Tags []tags.ResourceTag `json:"tags"`
 	// The enterprise project ID to which the private NAT gateway belongs.
 	EnterpriseProjectId string `json:"enterprise_project_id"`
 }
 
+// DownLinkVpc is an object that represents the VPC configuration to which private NAT gateway belongs.
```
Comment (Lance52259) [inline openstack/nat/v3/gateways/results.go:42]:
```suggestion
// DownLinkVpcResp is an object that represents the VPC configuration to which private NAT gateway belongs.
```

Comment (ruwenqiang123) [reply openstack/nat/v3/gateways/results.go]: done

**Pull request merged**